plugin-dev/Source/Sentry/Private/Android/Java/SentryBridgeJava.java

```diff
@@ -45,15 +45,24 @@ public void configure(SentryAndroidOptions options) {
 				options.setEnableAutoSessionTracking(enableAutoSessionTracking);
 				options.setSessionTrackingIntervalMillis(sessionTimeout);
 				options.setAttachStacktrace(enableStackTrace);
-			}
-		});
-
-		Sentry.configureScope(new ScopeCallback() {
-			@Override
-			public void run(@NonNull Scope scope) {
-				if(!gameLogPath.isEmpty()) {
-					scope.addAttachment(new Attachment(gameLogPath));
-				}
+				options.setBeforeSend(new SentryOptions.BeforeSendCallback() {
```

Comment:
I'm not entirely sure how that works underneath but can that callback be overwritten by users?

Reply:
We don't expose any APIs allowing users to do that, so it should be safe to set the `beforeSend` hook like this.

Reply:
Considering that we want to expose that API in the future, we could consider to add this as an eventprocessor/integration instead?

Reply:
Yeah, I think it should be feasible.